docs/example/override.yaml

```diff
@@ -12,16 +12,18 @@ image:
  name: terraform-enterprise
  tag: beta-1
```

Comment:
Should we change it to <vYYYYMM-#>

Reply:
updated!  also changed the registry location to point to the future prod release registry